site/using-weave/dynamically-attach-containers.md

```diff
@@ -37,6 +37,13 @@ specify `--rewrite-hosts` when running `weave attach`:
 
     host1$ weave attach --rewrite-hosts c1
 
+Weave Net will set the bridge side of the `veth` pair to enable hairpin mode when attaching a container.
```

Comment:
Yeah, good point. We need to change that. We want `hairpin=true` by default, do we not?